tests/ci/docker_compose.sh

```diff
@@ -7,6 +7,13 @@ docker pull $DOCKER_IMAGE
 
 docker-compose up -d
 
-sleep 120
+#check if nuoadmin is up
 
-docker exec "$(docker container ls -f name=nuoadmin -q)" nuocmd check database --db-name hockey --check-running --num-processes 2 --timeout 300
+while [ "$(docker container inspect -f '{{.State.Running}}' "$(docker ps -f name=nuoadmin -q)")" = "false" ]
+do
+    sleep 1
+    echo "Sleep for a second"
+done
+
+
+docker exec "$(docker container ls -f name=nuoadmin -q)" nuocmd check database --db-name hockey --check-running --num-processes 2 --timeout 300
```

Comment:
Nit: You use directly `docker exec <container name>`.

Reply:
I think the issue is that Docker Compose comes up with longer names for the containers. The name here is actually the hostname that gets supplied to the container, not the container name.